src/SFA.DAS.EmployerApprenticeshipsService.Application.UnitTests/Mappings/WhenAutoMapping.cs

```diff
@@ -15,7 +15,6 @@ public void ThenShouldUseValidConfiguration()
                 c.AddProfile<AccountMappings>();
                 c.AddProfile<AgreementMappings>();
                 c.AddProfile<LegalEntityMappings>();
-                c.AddProfile<MembershipMappings>();
```

Comment:
why has this been removed? are membershipmappings no longer needed?

Reply:
Not in EAS or Finance, they should now only exist in EmployerAccounts.